var/spack/repos/builtin/packages/py-pyside/package.py

```diff
@@ -84,14 +87,9 @@ def patch(self):
         # PySide can't find the Shiboken library, even though it comes
         # bundled with it and is installed in the same directory.
 
-        # PySide does not provide official support for
-        # Python 3.5, but it should work fine
-        filter_file(
-            "'Programming Language :: Python :: 3.4'",
-            "'Programming Language :: Python :: 3.4',\r\n        "
-            "'Programming Language :: Python :: 3.5'",
-            "setup.py",
-        )
+        # remove check for python version becaus Python 3.5 is not officially
+        # supported, but it should work fine
```

Comment:
```suggestion
        # Remove check for python version because the above patch adds support for newer versions
```

Reply:
You could even move it below the subprocess patch to make this clear

Reply:
Done